birdhouse/components/monitoring/config/proxy/conf.extra-service.d/monitoring.conf.template

```diff
@@ -0,0 +1,15 @@
+
+    location /grafana {
+        proxy_pass https://${PAVICS_FQDN_PUBLIC}${TWITCHER_PROTECTED_PATH}/grafana;
```

Comment:
We are not using the Verify trick?

Reply:
No, but we could. I don't see any reason not to and maybe that will help with the issues below (I'll look into that today)